cypress/e2e/signIn.cy.js

```diff
@@ -2,10 +2,37 @@
 
 describe('Sign In page', () => {
   beforeEach(() => {
+    cy.visit('/login');
+  });
+
+  it('Login with valid creds', () => {
```

Comment:
```suggestion
  it('should have an ability to log in user with valid credentials', () => {
```
Remember that test name should be the continuation of the describe. Fix everywhere